OrbitCore/OrbitModule.cpp

```diff
@@ -23,7 +25,26 @@
 #endif
 
 //-----------------------------------------------------------------------------
-Module::Module() { m_Pdb = std::make_shared<Pdb>(); }
+Module::Module(const std::string& file_name, uint64_t address_start,
+               uint64_t address_end) {
+  if (!Path::FileExists(file_name))
```

Comment:
I would always add {} for an if, even if it's a single line in the body.

Reply:
done